### PR TITLE
Feature/assignment 5

### PR DESCRIPTION
**Pull request by nikhil1-successive**

### CODE REVIEW

File: errors.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+There are a surprising number of HTTP status codes aside from the two or three you may have run into in the wild. They exist to represent the status of requests that browsers make to load websites, and they're split into five groups.
```
Comment (shahbaz-successive) [inline errors.md:1]: Add all error codes

File: src/middleware/tokenVerificationMiddleware.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+const tokenVerificationMiddleware = (req, res, next) => {
```
Comment (shahbaz-successive) [inline src/middleware/tokenVerificationMiddleware.js:1]: Remove this file

File: src/routes/userRoutes.js [hunk 3]
```diff
@@ -40,6 +40,9 @@ router.post('/login', customMiddleware, (req, res) => {
 router.get('/chainmiddleware', customMiddleware, authMiddleware, middleware1, middleware2, (req, res) => {
   res.send("Middleware Called")
 })
+router.get('/console', customMiddleware, (req, res) => {
```
Comment (shahbaz-successive) [inline src/routes/userRoutes.js:43]: Remove this route

File: src/middleware/tokenVerificationMiddleware.js (hunk 2 above)
Comment (nikhil1-successive) [reply src/middleware/tokenVerificationMiddleware.js]: done

File: src/routes/userRoutes.js (hunk 3 above)
Comment (nikhil1-successive) [reply src/routes/userRoutes.js]: done

File: src/routes/userRoutes.js [hunk 4]
```diff
@@ -60,4 +64,22 @@ router.get('/location', locationMiddleware, (req, res) => {
...
+    return res.send(result);
+  } catch (error) {
+    next(error);
+  }
+}));
+
+router.post('/params', validateParameters, (req, res) => {
+  res.json({
+    message: 'Success',
+    status: '400',
```
Comment (shahbaz-successive) [inline src/routes/userRoutes.js:80]: Code not is string

**Review by shahbaz-successive — APPROVED** [review]
👍

**Pull request merged**